packages/block-renderer/src/hooks/use-pattern-inline-css.tsx

```diff
@@ -0,0 +1,29 @@
+import { shuffle } from '@automattic/js-utils';
+import { useMemo } from 'react';
+import { MAX_BLOG_POSTS } from '../constants';
+
+// Using this object to memoize the pattern as a workaround
+const memoPatternOrder: { [ key: string ]: string } = {};
+
+const usePatternInlineCss = ( patternId: string ) => {
```

Comment:
Currently we add this inline CSS to literally every pattern. Would it be better if we first check if the `patternHtml` contains `is-layout-grid`, to save some bytes?